internal/config/local.go

```diff
@@ -0,0 +1,262 @@
+package config
```

Comment:
I tried to simplify a few places in this code after copying it from `influxdb`, please call out anywhere else that could benefit from a refactor.